lib/service/customer_support_service.dart

```diff
@@ -157,18 +176,58 @@ class CustomerSupportServiceImpl extends CustomerSupportService {
     processMessages();
   }
 
+  @override
+  Future<void> removeErrorMessage(String uuid, {bool isDelete = false}) async {
+    retryTime = 0;
+    final id = uuid.substring(0, 36);
+    errorMessages.remove(id);
+    if (isDelete) {
+      var msg = await _draftCustomerSupportDao.getDraft(id);
+      if (msg != null) {
+        final data = DraftCustomerSupportData.fromJson(jsonDecode(msg.data));
+        await _draftCustomerSupportDao.deleteDraft(msg);
+        if (msg.draftData.attachments != null && msg.draftData.attachments!.isNotEmpty) {
+          var draftData = msg.draftData;
+          String name = uuid.substring(36);
+          int index = draftData.attachments!.indexWhere((element) => element.fileName.contains(name));
```

Comment:
suggest using `removeWhere`

Reply:
or maybe using `firstWhereOrNull` if you need the object after that.
then `if (list.remove(object))` to execute the code with condition.